contributing/README.md

```diff
@@ -55,24 +55,24 @@ Beyond the official documentation there are endless possibilities for combining
 tools, platforms, languages, and products. By submitting a tutorial you can
 share your experience and help others who are solving similar problems.
 
-Community tutorials are stored in Markdown files on
-[GitHub](./samples/README.md) where they can be reviewed and edited by the
-community.
+Community tutorials are stored in Markdown files under the `community` folder
+[Community Samples](../community/samples/README.md).  These documents
+are contributed, reviewed, and maintained by the community.
 
-Please submit a Pull Request to the community sample directory under the Knative
-component that your tutorial highlights - [Serving](../community/samples/serving/),
-[Eventing](../community/samples/eventing/), or [Build](../community/samples/build/). A reviewer will
-be assigned to review your submission. They'll work with you to ensure your
-submission meets the [style guide](DOCS-CONTRIBUTING.md), but it helps if you
-follow it as you write your tutorial.
+Submit a Pull Request to the community sample directory under the Knative
+component folder that aligns with your document. For example, Knative Serving
+samples are under the `serving`. A reviewer will be assigned to review your 
+submission. They'll work with you to ensure your submission meets the 
+[style guide](./DOCS-CONTRIBUTING.md), but it helps if you follow it as you 
+write your tutorial.
 
```

Comment:
I know this isn't the intent of the PR, but this language makes it sound like the review is only about aligning to the style guide. Can we add more general language?

```suggestion
They'll work with you to ensure that your submission is clear, correct, and follows the style guide...
```